src/app/components/media/BulkActions.js

```diff
@@ -1,247 +1,39 @@
-      const message = this.props.page === 'trash' ? (
-        <FormattedMessage
-          id="bulkActions.movedRestoreSuccessfully"
-          defaultMessage="{count, plural, one {# item} other {# items}} moved from Trash to 'All'"
-          values={{
-            count: this.props.selectedMedia?.length,
-          }}
-          description="Banner displayed after items are moved successfully. 'All' here is the name of the default view in the workspace, which is localized under the id projectsComponent.allItems"
-        />
-      ) : (
-        <FormattedMessage
-          id="bulkActions.movedFromSpamSuccessfully"
-          defaultMessage="{count, plural, one {# item} other {# items}} moved from Spam to 'All'"
-          values={{
-            count: this.props.selectedMedia?.length,
-          }}
-          description="Banner displayed after items are moved successfully. 'All' here is the name of the default view in the workspace, which is localized under the id projectsComponent.allItems"
-        />
-      );
-      this.props.setFlashMessage(message, 'success');
-      this.props.onUnselectAll();
-    };
-
-    if (this.props.selectedMedia.length) {
-      Relay.Store.commitUpdate(
-        new BulkRestoreProjectMediaMutation({
-          ids: this.props.selectedMedia,
-          team: this.props.team,
-          archived_was: params.archived_was,
-        }),
-        { onSuccess, onFailure: this.fail },
-      );
-    }
-  }
-
-  handleArchive(archived) {
-    const onSuccess = () => {
-      const message = archived === CheckArchivedFlags.TRASHED ? (
-        <FormattedMessage
-          id="bulkActions.moveToTrashSuccessfully"
-          defaultMessage="{count, plural, one {# item} other {# items}} moved to Trash."
-          values={{
-            count: this.props.selectedMedia?.length,
-          }}
-          description="Message that appears when one or more items have been selected and moved to the trash."
-        />
-      ) : (
-        <FormattedMessage
-          id="bulkActions.moveToSpamSuccessfully"
-          defaultMessage="{count, plural, one {# item} other {# items}} moved to Spam."
-          values={{
-            count: this.props.selectedMedia?.length,
-          }}
-          description="Message that appears when one or more items have been selected and moved to spam."
-        />
-      );
-      this.props.setFlashMessage(message, 'success');
-      this.props.onUnselectAll();
-    };
-
-    if (this.props.selectedMedia.length) {
-      const mutation = new BulkArchiveProjectMediaMutation({
-        ids: this.props.selectedMedia,
-        team: this.props.team,
-        archived,
-      });
-      Relay.Store.commitUpdate(mutation, { onSuccess });
-    }
-  }
+  fail = () => {};
 
   render() {
     const {
-      page, team, selectedMedia,
+      page, team,
     } = this.props;
-    const disabled = selectedMedia.length === 0;
+    // const disabled = selectedMedia.length === 0;
```

Comment:
@danielevalverde what is the purpose of this? maybe just left in by accident?

Reply:
Thanks, @brianfleming. Yep, left by accident, deleted :)